src/msw/stdpaths.cpp

```diff
@@ -367,4 +367,10 @@ wxString wxStandardPathsWin16::GetUserConfigDir() const
     return wxGetHomeDir();
 }
 
+wxString wxStandardPaths::GetSharedLibrariesDir() const
+{
+    wxFileName fn( GetExecutablePath() );
+    return fn.GetPath() + wxFileName::GetPathSeparator();
```

Comment:
Have you seen `wxFileName::GetPathWithSep()`?
Also relevant in other places.

Reply:
@MaartenBent ,
Didnt know about that. Fixed. Thx.